packages/block-library/src/image/index.php

```diff
@@ -22,7 +22,7 @@ function render_block_core_image( $attributes, $content, $block ) {
 
 	$processor = new WP_HTML_Tag_Processor( $content );
 
-	if ( ! $processor->next_tag( 'img' ) || null === $processor->get_attribute( 'src' ) ) {
+	if ( ! $processor->next_tag( 'IMG' ) || null === $processor->get_attribute( 'src' ) ) {
```

Comment:
this isn't as important, but I carried it over from my other patch since I've tried to standardize tag names in the HTML API in uppercase, only because that's the form of the tag name that the API hands out